presto-spi/src/main/java/io/prestosql/spi/security/Identity.java

```diff
@@ -27,17 +27,24 @@
     private final String user;
     private final Optional<Principal> principal;
     private final Map<String, SelectedRole> roles;
+    private final Map<String, String> credentials;
```

Comment:
same comments like in `ConnectorIdentity`

Reply:
Also rename this to `extraCredentials`